### ISSUE TITLE
Add ability to use http for metrics

### ISSUE THREAD

**Issue by JamieDanielson** [honeycombio/honeycomb-opentelemetry-java#293]
**Is your feature request related to a problem? Please describe.**

In #283 we added ability to use http protocol for traces, but it will not work for metrics because metrics will have a different signal-specific path (`v1/metrics`).

**Describe the solution you'd like**

We should add configuration options for [signal-specific protocols](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specify-protocol) as well as the appended HTTP path for metrics - similar to what we did for endpoints:

- `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL`
- `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL`

**Comment by cartermp** [comment]
Would this also require husky work?

**Comment by JamieDanielson** [comment]
> Would this also require husky work?

No, we can ingest everything appropriately through Husky, but it's not being set right here. The problem is that our SDK sets the endpoint to honeycomb, and needs the signal-specific path appended when using a signal-specific endpoint. We added `v1/traces` for the generic `OTEL_EXPORTER_OTLP_PROTOCOL`, but that means metrics would also try to go to `v1/traces`.

**Comment by MikeGoldsmith** [comment]
Shepherd does not yet expose a metrics HTTP endpoint, it only support gRPC for now.

**Comment by JamieDanielson** [comment]
Oops, thank you @MikeGoldsmith ! @cartermp my bad, we would have to make some changes for metrics! 

With this in mind... it seems like we should set only the `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL` (instead of the generic `OTEL_EXPORTER_OTLP_PROTOCOL` to ensure metrics can still be sent over gRPC? Otherwise the ability to use http is only available for those not sending metrics.
